### PR TITLE
Removed white bar at bottom

### PR DESCRIPTION
**Pull request by antoinztte**
## Overview

- Removed the white bar so that the entire contents of individual eatery pages are shown (based on feedback from design)

## Changes Made

- Override viewDidLayoutSubviews func so that the scroll view fills the entire view

## Test Coverage

- Tested on iPhone 14 (all sizes).

## Related PRs or Issues

- Related to PR #9 (and Jennifer's feedback)

## Screenshots

<details>

</details>

### CODE REVIEW

**Review by NgaiJustin — APPROVED** [review]
👍

File: Eatery Blue/Eatery Blue/UI/EateryScreen/EateryPageViewController.swift [hunk 1]
```diff
@@ -35,6 +35,15 @@ class EateryPageViewController: UIPageViewController {
         setUpPages()
     }
     
+    override func viewDidLayoutSubviews() {
+        for subview in view.subviews {
```
Comment (ReadeP) [inline Eatery Blue/Eatery Blue/UI/EateryScreen/EateryPageViewController.swift:39]: use `view.subviews.forEach { ...` It's more Swifty 💃

File: Eatery Blue/Eatery Blue/UI/EateryScreen/EateryPageViewController.swift [hunk 2]
```diff
@@ -35,6 +35,15 @@ class EateryPageViewController: UIPageViewController {
...
+            if subview is UIScrollView {
+                subview.frame = self.view.bounds
```
Comment (ReadeP) [inline Eatery Blue/Eatery Blue/UI/EateryScreen/EateryPageViewController.swift:41]: You don't need `self` here

**Review by ReadeP — APPROVED** [review]
🔥🔥🔥

**Pull request merged**